react/resources/cljsjs/react/common/react.ext.js

```diff
@@ -245,8 +245,10 @@ React.Component.prototype.componentWillUnmount = function() {};
 
 /**
  * @protected
+ * @param {Object} error
+ * @param {Object} info
  */
-React.Component.prototype.componentDidCatch = function() {};
+React.Component.prototype.componentDidCatch = function() {error, info};
```

Comment:
The names are now inside the function body instead of the parameter list.

Reply:
I've fixed the placement. Sorry for the stupid mistake.